### DISCUSSION TITLE
Welcome to twitch-tui Discussions!

### DISCUSSION THREAD

**Discussion by Xithrius** [Xithrius/twitch-tui#39] [General]
## 👋 Greetings!

In this discussion area, you're welcome to ask questions, suggest ideas, and in general communicate with the developers of this project.

**Comment by balroggg** [comment]
May be add option to select channel to open?

- use clap crate or other tool to parse cli arguments and override config channel
- open new channel in main window
- startup window with channel selection, like vim startify(open empty channel and then select something)

Any thoughts?

**Reply by Xithrius** [reply]
I've been thinking about adding similar ideas for a while, just never got to them due to other issues. Hopefully I can start on them soon.

**Reply by balroggg** [reply]
With new clap release 3.0 they deprecate runtime parsing yaml config, should we need to rewrite this part or freeze dependency or something else?

**Reply by Xithrius** [reply]
We can just stay on not clap 3.0 for now.

**Comment by balroggg** [comment]
Any plans/thoughts for support twitch emotes? Maybe support free emotes from https://betterttv.com.

**Reply by Xithrius** [reply]
Supporting Twitch emotes is going to be difficult to support, due to specific channels having their own emotes, along with user-owned emotes. 

Might do in the future, most definitely after 2.0 is released.

**Comment by balroggg** [comment]
Do we need support of twitch badges? I can make a PR for this.

**Reply by Xithrius** [reply]
I'm doing a pretty large PR for this repo at this current moment (2.0), so I'd hold off on this until it's merged.

But I do approve of it happening. Can you make an issue for it?

Thanks!
